code/modules/surgery/organs/heart.dm

```diff
@@ -126,18 +127,18 @@
 
 /obj/item/organ/internal/heart/cursed/on_life(delta_time, times_fired)
 	if(world.time > (last_pump + pump_delay))
-		if(ishuman(owner) && owner.client) //While this entire item exists to make people suffer, they can't control disconnects.
+		if(ishuman(owner) && owner.client && owner.client?.lastping < 250) //While this entire item exists to make people suffer, they can't control disconnects.
```

Comment:
my ping is consistently over 250, which means I can use this item without any ill effects whatsoever

this is simply not a workable solution.